api/src/main/resources/org/openmrs/module/mirebalaisreports/sql/mexicoSisDiagnoses.sql

```diff
@@ -1,14 +1,20 @@
-# ins, value_coded, prev_visit
 SELECT
+    CONCAT(DATE(e.encounter_datetime), " ", TIME(e.encounter_datetime)) "Fecha",
+    CONCAT(given_name, " ", family_name) "Nombre",
     ROUND(DATEDIFF(e.encounter_datetime, pr.birthdate)/365.25, 1) "Edad",
     pr.gender "Genero",
     IF(ins.value_coded = seguro_popular.concept_id, '1', '') "SPSS",
-    IF(IFNULL(prev_visit.visit_id, 'null') = 'null', '1', '') "1a visita",
+    '' "1a vez",  #TODO MEX-410
```

Comment:
It'll be easiest to leave this dummy column here, then fix the SQL once there's data to support it. This will also allow doctors to manually input the data in the meantime (for which the date and name are intended to help)